api/graphql/schema.graphql

```diff
@@ -1953,6 +1963,13 @@ type Mutation {
     @update
     @guard
     @canFind(ability: "updatePublished", find: "id")
+  togglePoolUserBookmark(
+    userId: UUID! @rename(attribute: "user_id")
+    poolId: UUID! @rename(attribute: "pool_id")
+  ): Pool
+    @guard
+    @canFind(ability: "view", find: "pool_id", model: "Pool", injectArgs: true)
```

Comment:
This allows anyone to update anyone else's bookmark which we probably don't want. I think it is reasonable for someone to only be able to update their own bookmarks.

I think that can be easily done like so below using `inject`, where the acting/auth context object feeds into the `args.user_id` value

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/2283baa7-6d26-438e-9172-c0b00b652155)